make testlibgap fails when GAP is compiled with Julia

When the Julia GC is used, `make testlibgap` fails with the following error:
```
make testlibgap
   LINK    testlibgap                                                                                                                                                                                                               
./test-libgap -A -l . -m 32m -q -T --nointeract > basic.out                                                                                                                                                                         
gap/.libs/lt-test-libgap: error while loading shared libraries: libjulia.so.1: cannot open shared object file: No such file or directory                                                 
make: *** [Makefile.rules:995: testlibgap] Error 127 
```
GAP Banner:
```
 ┌───────┐   GAP 4.8.8-6161-gc8fb501-dirty of today                                                                                                                                                                                 
 │  GAP  │   https://www.gap-system.org                                                                                                                                                                                             
 └───────┘   Architecture: x86_64-pc-linux-gnu-julia64-kv6                                                                                                                                                                          
 Configuration:  gmp 6.1.2, Julia GC, Julia 1.1.0-DEV, readline  
```

--- a/Makefile.rules
+++ b/Makefile.rules
@@ -993,11 +993,11 @@ testbugfix: all
           ReadGapRoot( "tst/testbugfix.g" );' | $(TESTGAP) | \
             tee `date -u +dev/log/testbugfix2_%Y-%m-%d-%H-%M` )
 
-LIBGAPTESTS := $(addprefix tst/testlibgap/,basic wsload wscreate)
+LIBGAPTESTS := $(addprefix tst/testlibgap/,basic wscreate wsload)
 
 # run a test in tst/testlibgap
 tst/testlibgap/%: obj/tst/testlibgap/%.lo obj/tst/testlibgap/common.lo libgap.la
-	$(QUIET_LINK)$(LINK) $(GAP_LDFLAGS) $^ -o $@
+	$(QUIET_LINK)$(LINK) $(GAP_LDFLAGS) $^ $(GAP_LIBS) -o $@
 
 clean: clean-testlibgap
 clean-testlibgap:
@@ -1005,12 +1005,10 @@ clean-testlibgap:
 
 # run all the tests in tst/testlibgap
 testlibgap: ${LIBGAPTESTS}
-	$(foreach v,$^,                                              \
-	      $(v) -A -l $(top_srcdir) -q -T --nointeract >$(v).out; \
-	      diff $(top_srcdir)/$(v).expect $(v).out; )
-
-# this is to make sure wscreate is run before wsload : order-only dependence.
-tst/testlibgap/wsload: | tst/testlibgap/wscreate
+	$(foreach v,$^, \
+		echo "Running $(v) ..." ; \
+		$(v) -A -l $(top_srcdir) -q -T --nointeract >$(v).out ; \
+		diff $(top_srcdir)/$(v).expect $(v).out ; )
 
 .PHONY: testlibgap
 

